dashboard/models.py

```diff
@@ -152,13 +169,17 @@ def get_supported_courses(self):
 
 
 class CourseManager(models.Manager):
-    def get_queryset(self):
+    def get_queryset(self) -> QueryType:
```

Comment:
Can you explain the use of `QueryType` here? I'm seeing `<class 'dashboard.models.CourseQuerySet'>` as the returned type, but perhaps there is some advantage to using the `django_hint` classes. It would also be nice to be consistent with `UserDefaultManager`, where you are just using the model we defined (`UserDefaultQuerySet`).